docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1095 @@
+  3. **Subordinate Statement Removal**: The Federation Authority MUST completely remove the Entity's Subordinate Statement from the Federation Registry to prevent any trust relationship validation.
+  4. **Entity Configuration Deactivation**: The Entity MUST deactivate its Entity Configuration. The Entity MAY either:
+
+     a. Remove the Entity Configuration completely from the ``/.well-known/openid-federation`` endpoint (returning HTTP 404), OR
+     b. Keep the Entity Configuration available but MUST ensure it remains expired (with ``exp`` claim in the past) and MUST NOT update it with fresh timestamps.
+
+  5. **Registry Status Update**: The Entity SHOULD verify removal from Federation Registry, also verifying the Trust Mark status returns ``{"active": false}`` from ``/trust_mark_status`` endpoint. 
+
+Non-normative example of certificate revocation request following RFC 3280 format:
+
+.. code-block:: text
+
+   Certificate Revocation Request:
+   Subject: CN=credentials.example.gov, OU=Digital Credentials, O=Example Organization, L=Roma, ST=Lazio, C=IT, emailAddress=technical@credentials.example.gov
+   Certificate Serial Number: 987654321
+   Revocation Reason: cessation_of_operation (5)
+   Revocation Date: 2025-12-31T23:59:59Z
+
+   Request signed with Federation Entity Private Key corresponding to:
+   Public Key Algorithm: id-ecPublicKey
+   ASN1 OID: prime256v1
+   NIST CURVE: P-256
+   Key ID: NsXymfIILEPR5Y0t
+
+   Note: The CRR MUST be signed with the same private key that corresponds to the
+   certificate being revoked to authenticate the revocation request.
+
+Example CRR in DER format (Base64 encoded):
+
+.. code-block:: text
+
+   -----BEGIN CERTIFICATE REVOCATION REQUEST-----
+   MIIBVjCB/wIBADBpMQswCQYDVQQGEwJJVDEOMAwGA1UECAwFTGF6aW8xDTALBgNV
+   BAcMBFJvbWExGjAYBgNVBAoMEUV4YW1wbGUgT3JnYW5pemF0aW9uMR8wHQYDVQQD
+   DBZjcmVkZW50aWFscy5leGFtcGxlLmdvdjBZMBMGByqGSM49AgEGCCqGSM49AwEH
+   A0IABIBgZ4HBgUCNXwY5LJSlKzm7gXY4FApFJCj91Gpb1K9GEIouTq2X3L0K64Iq
+   0ob4l_gslT14644zbYXYF-xmw7aPdlbMuw3T1URwI4nafMtKrYwDQYJKoZIhvcNAQ
+   kEAwIJrRLl1VR987654321gBgJKwYBBQUHAgEWHGh0dHBzOi8vZXhhbXBsZS5vcmcv
+   cG9saWN5MAoGCCqGSM49BAMCA0gAMEUCIQC9h3Y6hFgd7zUzZyBrQ3jJ8HmVF2Qa
+   -----END CERTIFICATE REVOCATION REQUEST-----
+
+Supervisory Body Removal - Technical Implementation
+""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+  1. **Emergency Certificate Revocation**: The Federation Authority MUST immediately revoke certificates with appropriate reason code (e.g., "Key Compromise", "Cessation of Operation").
+  2. **CRL Emergency Update**: The Trust Anchor MUST publish updated CRL within emergency timeframe.
+  3. **Subordinate Statement Removal**: The Federation Authority MUST immediately and completely remove the Entity's Subordinate Statement from all federation endpoints.
```

Comment:
instead of duplicating the same informations and details in multiple part of the document, we could write these in an RST file and include it in each section

@fmarino-ipzs wdyt?